api/userTermsAndConditions/index.ts

```diff
@@ -0,0 +1,63 @@
+import * as express from 'express'
+import { config } from '../config'
+import { application } from '../config/application.config'
+import { GetUserAcceptTandCResponse, PostUserAcceptTandCResponse } from '../interface/userAcceptTandCResponse'
+import { http } from '../lib/http'
+import { isUserTandCPostSuccessful } from '../lib/util'
+import { getUserTermsAndConditionsUrl, postUserTermsAndConditionsUrl } from './userTermsAndConditionsUtil'
+
+export async function getUserTermsAndConditions(req: express.Request, res: express.Response) {
+    let errReport: any
+    if (!req.params.userId) {
+        errReport = {
+            apiError: 'UserId is missing',
+            apiStatusCode: '400',
+            message: 'User Terms and Conditions route error',
+        }
+        res.status(400).send(errReport)
+    }
+    try {
+        const url = getUserTermsAndConditionsUrl(config.services.termsAndConditions, req.params.userId, application.idamClient)
+        const response = await http.get(url)
+        const userTandCResponse = response.data as GetUserAcceptTandCResponse
+        res.send(userTandCResponse.accepted)
+    } catch (error) {
+        // we get a 404 if the user has not agreed to Terms and conditions
+        if (error.status === 404) {
+            res.send(false)
```

Comment:
for readability improvements, we should have a neum/constant file that stores all the server response codes.

Reply:
A constant file for http status codes? I feel that's a bit over the top. I agree that we shouldn't use magic numbers, but http status codes are not magic numbers.